docs/MultipleInstances.md

```diff
@@ -0,0 +1,10 @@
+# Running multiple instances
+
+If you want to setup multiple JackTrip instances on single machine, you have to use _different JACK servers_. Set `--name` flag for `jackd` flag and accordingly set `JACK_DEFAULT_SERVER` environment variable for JackTrip instances.
```

Comment:
You can run multiple JackTrip instances with the same JACK server. What was your use case that you tried this? One use case would be to use different audio interfaces for every jacktrip instance.

Reply:
Yep, I should be more clear about the purpose of running different JACK servers. Should I change main agenda to "Using JACK server with non-default name"?

Reply:
The idea of starting with real use cases is good. "Using JACK server with non-default name" is more on the solution side.
So starting with your use case of isolating JackTrip enviroments and the use case I mentioned about using different audio interfaces would be nice.
The heading could be **Using multiple JACK servers**. Maybe @dyfer has an opinion here?

Reply:
I think this document should first show running multiple jack instances with a single jack server first and also be explicit that this is only with the Jack backend, as opposed to the RtAudio backend. 
Then a subsection with a heading **Using multiple JACK servers** could explain that specific use case.

Also, I don't think we should be adding `venv` to the `.gitignore` - having python code inside this repository is probably not the best use case.

Finally, the PR should be aimed at `dev`.

Thanks!